edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/EdgeDeploymentController.cs

```diff
@@ -232,37 +233,25 @@ async Task ManagePvcs(V1PersistentVolumeClaimList existing, IEnumerable<V1Persis
                     });
             await Task.WhenAll(updatingTask);
 
-            try
-            {
-                // Remove all PVCs that are not in the desired list, and are labeled (created by Agent)
-                var removingTasks = diff.Removed
-                    .Select(
-                        name =>
-                        {
-                            Events.DeletePvc(name);
-                            return this.client.DeleteNamespacedPersistentVolumeClaimAsync(name, this.deviceNamespace);
-                        });
-                await Task.WhenAll(removingTasks);
+            // Remove all PVCs that are not in the desired list, and are labeled (created by Agent)
+            var removingTasks = diff.Removed
+                .Select(
+                    name =>
+                    {
+                        Events.DeletePvc(name);
+                        return this.client.DeleteNamespacedPersistentVolumeClaimAsync(name, this.deviceNamespace);
+                    });
+            await Task.WhenAll(removingTasks);
 
-                // Create all new desired PVCs.
-                var addingTasks = diff.Added
-                    .Select(
-                        pvc =>
-                        {
-                            Events.CreatePvc(pvc);
-                            return this.client.CreateNamespacedPersistentVolumeClaimAsync(pvc, this.deviceNamespace);
-                        });
-                await Task.WhenAll(addingTasks);
-            }
-            catch (HttpOperationException ex)
```

Comment:
Have you tested this w/storage class PVC requests?  The comment in the removed code may still apply.

Reply:
Yeah... seems suspicious that try-catch here was added here intentionally. Or the error described in comment section is caused by lack of permissions?

Reply:
I may have misinterpreted the error I received while testing, but I'd like confirmation.